{"code":"ER_ACCESS_DENIED_ERROR","errno":1045,"level":"error","message":"MySQL connection error: Access denied for user 'root'@'localhost' (using password: YES)","sqlState":"28000","stack":"Error: Access denied for user 'root'@'localhost' (using password: YES)\n    at Object.createConnectionPromise [as createConnection] (/Users/rameshkashyap/Desktop/secure_node_project/node_modules/mysql2/promise.js:18:31)\n    at connectDB (/Users/rameshkashyap/Desktop/secure_node_project/server.js:37:40)\n    at Object.<anonymous> (/Users/rameshkashyap/Desktop/secure_node_project/server.js:45:1)\n    at Module._compile (node:internal/modules/cjs/loader:1469:14)\n    at Module._extensions..js (node:internal/modules/cjs/loader:1548:10)\n    at Module.load (node:internal/modules/cjs/loader:1288:32)\n    at Module._load (node:internal/modules/cjs/loader:1104:12)\n    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:173:12)\n    at node:internal/main/run_main_module:28:49"}
2025-04-05 12:40:11 ERROR: UPI Callback Error: ReferenceError: Income is not defined
    at dynamicUpiCallback (/home/hypermes/api.hypermesh.io/hypermesh-node/src/controllers/InvestController.js:166:14)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
